### DISCUSSION TITLE
[Feature] Make tabby as drop-down console like another console client

### DISCUSSION THREAD

**Discussion by WhiteBite** [Eugeny/tabby#4830] [General]
Make tabby as drop-down console like another console client
![image](https://user-images.githubusercontent.com/19576772/138695381-69b4cc44-4926-4a0d-a9e2-c5634fa494c7.png)
Like this.

**Comment by Eugeny** [comment]
You can enable this (docking / sticky / "quake" mode) in Settings -> Window:
![image](https://user-images.githubusercontent.com/161476/138695900-d9fbf0b7-3752-49a9-8270-ce98ef9977d7.png)

**Reply by WhiteBite** [reply]
How to make it collapse and unfold by keyboard shortcut?

**Reply by Eugeny** [reply]
Settings - Hotkeys - Toggle Window

**Reply by WhiteBite** [reply]
![image](https://user-images.githubusercontent.com/19576772/138837586-0c25ae4f-79ef-4a2d-aaba-3be81532f353.png)
Found a bug, if the panel is on top, it overlaps(

**Reply by Eugeny** [reply]
Thanks! Could you please post it as a separate issue?

**Reply by WhiteBite** [reply]
This is corrected by manually dragging the window. Thank you! Okey

**Reply by WhiteBite** [reply]
I fooled you, when I toggle windows, the tabby window appears as much as possible at the top. This cannot be fixed by hand(
I make issue
